trollflow2/plugins/__init__.py

```diff
@@ -216,6 +217,23 @@ def prepared_filename(fmat, renames):
         yield orig_filename
 
 
+def format_decoration(fmat,  fmat_config):
+    """
+    Format decoration in fmt_config according to fmat.
+    """
```

Comment:
```suggestion
    """Format decoration in fmt_config according to fmat."""
```

Reply:
This docstring could also be a bit more descriptive.

Reply:
Maybe something like `Format decoration text using template given in fmt_config with key-value pairs in fmat.`?

Reply:
Thanks for suggestions, I implemented them.